sdk/extensions/chronopost/karrio/providers/chronopost/shipment/create.py

```diff
@@ -0,0 +1,204 @@
+from datetime import datetime
+import typing
+from chronopost_lib.shippingservice import (
+    customerValue,
+    recipientValue,
+    refValue,
+    resultMultiParcelValue,
+    shippingMultiParcelV5,
+    shipperValue,
+    skybillParamsValue,
+    skybillValue,
+)
+import typing
+import karrio.lib as lib
+import karrio.core.models as models
+import karrio.providers.chronopost.error as provider_error
+import karrio.providers.chronopost.utils as provider_utils
+import karrio.providers.chronopost.units as provider_units
+
+
+def parse_shipment_response(
+    response: lib.Element, settings: provider_utils.Settings
+) -> typing.Tuple[models.ShipmentDetails, typing.List[models.Message]]:
+    errors = provider_error.parse_error_response(response, settings)
+    shipment = (
+        _extract_details(response, settings)
+        if lib.find_element("resultMultiParcelValue", response, first=True) is not None
+        else None
+    )
```

Comment:
```suggestion
    shipment_node = lib.find_element("resultMultiParcelValue", response, first=True)
    shipment = (
        _extract_details(result, settings)
        if shipment_node is not None else None
    )
```

I think that is the reason why the shipment parsing is failing.

Reply:
I also changed `shipment = lib.to_object(resultMultiParcelValue, response)` to ` shipment = lib.find_element("resultMultiParcelValue", response, resultMultiParcelValue, first=True)` in `_extract_details()` and that fixed it.

Reply:
🤔 I made a mistake actually. there was no `result` variable it was supposed to be `shipment_nodes`.

Once that is done, I don't think you need to do this change: 

> I also changed `shipment = lib.to_object(resultMultiParcelValue, response)` to ` shipment = lib.find_element("resultMultiParcelValue", response, resultMultiParcelValue, first=True)` in `_extract_details()` and that fixed it.